isp/rules.v4

```diff
@@ -0,0 +1,35 @@
+# Generated by iptables-save v1.6.0 on Mon Jun  4 19:43:58 2018
+*nat
+:PREROUTING ACCEPT [5:1622]
+:INPUT ACCEPT [3:180]
+:OUTPUT ACCEPT [4:308]
+:POSTROUTING ACCEPT [1:96]
+-A POSTROUTING -o wlan0 -j MASQUERADE
+COMMIT
+# Completed on Mon Jun  4 19:43:58 2018
+# Generated by iptables-save v1.6.0 on Mon Jun  4 19:43:58 2018
+*filter
+:INPUT ACCEPT [2:160]
+:FORWARD DROP [0:0]
+:OUTPUT ACCEPT [1:96]
+-A INPUT -i lo -j ACCEPT
+-A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -i eth0 -p udp -m udp --dport 67 -j ACCEPT
+-A INPUT -i eth0 -p tcp -m tcp --dport 22 -j ACCEPT
+-A INPUT -i eth0 -p udp -m udp --dport 53 -j ACCEPT
+-A INPUT -i eth0 -p tcp -m tcp --dport 53 -j ACCEPT
+-A INPUT -i eth0 -p icmp -j ACCEPT
+-A INPUT -i wlan0 -p icmp -m icmp --icmp-type 8 -j ACCEPT
+-A INPUT -i wlan0 -p icmp -m icmp --icmp-type 0 -j ACCEPT
+-A FORWARD ! -i wlan0 -j ACCEPT
```

Comment:
Accept any traffic that is forwarded between ethernet interfaces. In other words, anything that is not coming from wlan0.